cep-XXXX.md

```diff
@@ -0,0 +1,267 @@
+# CEP XXXX - Specification of <code>environment.yml</code> input files
+
+<table>
+<tr><td> Title </td><td> CEP XXXX - Specification of <code>environment.yml</code> input files </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Jaime Rodríguez-Guerra &lt;jaime.rogue@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> June 4, 2024 </td></tr>
+<tr><td> Updated </td><td> March 14, 2025 </td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda/ceps/pull/81 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CEP standardizes the format of `environment.yml` input files for conda clients.
+
+## Motivation
+
+The motivation of this CEP is merely informative. It describes the details of an existing file
+format.
+
+## Nomenclature
+
+The `environment.yml` file format was introduced by `conda env`. It has been traditionally referred
+to as "the YAML input file", but other YAML files exist in the ecosystem now. To avoid confusion,
+in this CEP we will only use the term `environment.yml`. Do note that the actual filename of an
+input file using this format might be a different one (e.g. `env.yml`).
+
+## Specification
+
+`environment.yml` files are YAML documents that encode the necessary information to create a new
+conda environment. The core standard recognizes five top-level keys:
+
+- `name`: The preferred name for the environment.
+- `prefix`: The preferred full path to the environment. Often ignored.
+- `dependencies`: List of package specifications that MUST be installed in the new environment.
+- `channels`: conda channels that will be used to resolve the dependencies.
+- `variables`: Environment variables that SHOULD be added to the `conda-meta/state` file in the
+  resulting environment.
+
+The file name is not standardized, but the extension MUST be `yml` or `yaml`.
+
+Notably, this version of the specification does not feature a `version` (or equivalent) top-level
+key. Future revisions MAY include such a field. As such, the version information of this revision
+is omitted and considered to be `1`.
```

Comment:
The community has talked about a [v2](https://github.com/conda/conda/issues/11341) format for a while, so I'm calling this one v1, but open for suggestions.

Reply:
This is going to confuse everything since recipe formats go from v0 to v1. Let's call this one `v0`.

Reply:
I've thought more about this and after finding some examples, it sounds like `v0` for `meta.yaml` is more the exception, not the norm. See rationale in https://github.com/conda/ceps/pull/119#issuecomment-2732831737 and following comments.